_ext/eclipse-wtp/gradle.properties

```diff
@@ -19,8 +13,7 @@ VER_ECLISPE_EMF=[2.16.0,3.0.0[
 VER_ECLIPSE_OSGI_SERVICES=[3.8.0,4.0.0[
 VER_ECLIPSE_FILE_BUFFERS=[3.6.700,4.0.0[
 VER_ECLISPE_JFACE=[3.15.300,4.0.0[
-VER_ECLISPE_EFS=[1.7.500,2.0.0[
+#VER_ECLISPE_EFS=[1.7.500,2.0.0[
+#TODO: 1.7.600 changed bundle activation. Code adaptation required.
+VER_ECLISPE_EFS=1.7.500
```

Comment:
No comment needed, just noting the change.

Reply:
Brief comment :wink: 
Change allows check of this proposal by `./gradlew --write-locks -Pcom.diffplug.spotless.include_ext=true build`